### ISSUE TITLE
Fullbright pipe in cruz map with vertex lighting

### ISSUE THREAD

**Issue by illwieckz** [DaemonEngine/Daemon#1027]
Fullbright pipe in cruz map with vertex lighting:

Light mapping:

![unvanquished_2024-01-13_162227_000](https://github.com/DaemonEngine/Daemon/assets/2311649/7c36eea6-bdcf-410a-9ef9-2b0ea0904028)

Vertex lighting:

![unvanquished_2024-01-13_162254_000](https://github.com/DaemonEngine/Daemon/assets/2311649/eebef0e4-3127-48ef-9896-d6fafa4555db)

I don't know yet if it's a bug, it's possible this is not a bug if vertex light is fullbright there. 

The test scene is the default spectator scene.

map|viewpos
-|-
cruz|-1462 -651 164 36 0

**Comment by illwieckz** [comment]
Here is the material:

```c
textures/conveyor/p_01_s
{
	qer_editorimage textures/conveyor/p_01
//	q3map_surfacelight 50
	surfaceparm nomarks
	{
		map textures/conveyor/p_01
	}
	{
		map $lightmap
		blendfunc filter
		tcGen lightmap
	}
	{
		map textures/conveyor/envmap
		tcGen environment
		blendfunc gl_dst_color gl_src_alpha
	}
}
```

**Comment by illwieckz** [comment]
The `tcGen lightmap` line should not be there, it is superfluous, I guess it makes the stage wrongly detected as a light style.

**Comment by illwieckz** [comment]
I added a commit to add a debug cvar named `r_showVertexColors` to render the vertex colors used for vertex lighting:

![unvanquished_2024-01-13_172100_000](https://github.com/DaemonEngine/Daemon/assets/2311649/6660410d-18c7-4f6e-bab5-7369f66b0040)

![unvanquished_2024-01-13_172014_000](https://github.com/DaemonEngine/Daemon/assets/2311649/f73a9cd3-5f50-4705-a39a-92f10d15aef1)

**Comment by illwieckz** [comment]
Fixed by #1026:

- https://github.com/DaemonEngine/Daemon/pull/1026
